modules/genai-ecosystem/pages/apoc-genai.adoc

```diff
@@ -0,0 +1,100 @@
+= APOC GenAI Procedures
+:slug: apoc-genai
+:author: 
+:category: genai-ecosystem
+:tags: embeddings, chat, completion, image generation, genai, query generation, schema explanation
+:neo4j-versions: 5.11+
+:page-pagination:
+:page-product: apoc-genai
+
+The APOC Extended procedure library integrates with various AI/ML platforms to provide a set of useful utilities for quickly demonstrate GenAI features.
+The library includes procedures for generating texts, images, and vector embedding.
+It also allows the generation and execution of Cypher queries from natural language as well as schema explanation.
+
+APOC Extended integrates with 
+
+* https://neo4j.com/labs/apoc/5/ml/openai/[(Azure) OpenAI^], 
+* https://neo4j.com/labs/apoc/5/ml/bedrock/[AWS Bedrock^], 
+* https://neo4j.com/labs/apoc/5/ml/vertexai/[Google Vertex AI^], 
+* IBM WatsonX 
+* and other ML platforms that have OpenAI compatible APIs.
```

Comment:
Should this be Open A**P**I compatible APIs?

Reply:
No OpenAI, there are a lot of LLM services that have the same REST API as OpenAI so most of the tools work by just changing the base URL.